FusionIIIT/applications/online_cms/models.py

```diff
@@ -187,11 +189,37 @@ def __str__(self):
         return '{} - {} - {}'.format(self.pk, self.course_id, self.assignment_name)
 
 #details of the solution uploaded by the student
-class StudentAssignment(models.Model):
+def assignment_submit_name(instance, filename):
+    name, ext = filename.split('.')
+    # obj=Curriculum.objects.get(course_id=instance.course_id)
+    course_code=instance.course_code
+    assignmentName=instance.assignment_id.assignment_name
+    file_path = 'online_cms/{course_id}/assi/{assignmentName}/{fileName}.{ext}'.format(
+         course_id=course_code,assignmentName=assignmentName, fileName=name, ext=ext) 
+    return file_path
+# class StudentAssignment(models.Model):
+#     student_id = models.ForeignKey(Student, on_delete=models.CASCADE)
+#     assignment_id = models.ForeignKey(CourseAssignment, on_delete=models.CASCADE)
+#     upload_time = models.DateTimeField(auto_now=True)
+#     # upload_url = models.TextField(max_length=200)
+#     doc = models.FileField(upload_to=assignment_submit_name, null=True, blank=True)
+#     score = models.IntegerField(null=True)        #score is submitted by faculty 
+#     feedback = models.CharField(max_length=100, null=True)  #feedback by the faculty for the solution of the assignment submitted
+#     assign_name = models.CharField(max_length=100) 
+
+#     def __str__(self):
+#         return '{} - {} - {} - {} - {}'.format(
+#                 self.pk, self.student_id,
+#                 self.assignment_id, self.score,
+#                 self.feedback)
```

Comment:
Kindly remove the comment which is not needed.

Reply:
Will remove it after i am sure that any other module isn't using it